deployment/clouddeploy/deployer.go

```diff
@@ -543,6 +831,80 @@ func (d *Deployer) UpdateClusterExpiry(ctx context.Context, clusterID string, ne
 	return nil
 }
 
+func isNotEqual(spec1, spec2 capellacontrol.UpdateClusterSpecsRequest_Spec) bool {
```

Comment:
nit: Generally in Go, you would use a 'Compare' (possible a method of the UpdateClusterSpecsRequest, or maybe just a global CompareUpdateClusterSpecsRequest) method which returns an int indicating whether they are the same, and then "isNotEqual" would be checking if its non-zero.  Example: https://github.com/couchbaselabs/cbdinocluster/blob/master/deployment/dockerdeploy/imageprovider.go#L27

Reply:
Updated